Dockerfile-local

```diff
@@ -14,6 +16,7 @@ RUN gem install jekyll
 ADD Gemfile /tmp/Gemfile 
 ADD Gemfile.lock /tmp/Gemfile.lock
 
+RUN npm install pug -g
```

Comment:
I would put this before any `ADD` so docker have a chance to cache this layer. 
Could be right after `RUN apt-get install -y nodejs`